Content.Server/Chemistry/ReagentEffects/Polymorph.cs

```diff
@@ -0,0 +1,37 @@
+using Content.Server.Mind.Components;
+using Content.Server.Polymorph.Components;
+using Content.Server.Polymorph.Systems;
+using Content.Shared.Chemistry.Reagent;
+using Content.Shared.Polymorph;
+using Robust.Shared.Prototypes;
+
+namespace Content.Server.Chemistry.ReagentEffects;
+
+public sealed partial class Polymorph : ReagentEffect
+{
+    /// <summary>
+    ///     What polymorph prototype is used on effect
+    /// </summary>
+    [DataField("prototype")] public string PolymorphPrototype { get; set; }
+
+    protected override string? ReagentEffectGuidebookText(IPrototypeManager prototype, IEntitySystemManager entSys)
+        => Loc.GetString("reagent-effect-guidebook-make-polymorph",
+            ("chance", Probability), ("protoname", prototype.Index<PolymorphPrototype>(PolymorphPrototype)));
```

Comment:
Using the polymorph prototype, get the entity it transforms into, index that, and get the name. Don't use prototype Ids in player-facing contexts.

Reply:
Done